spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/core/convert/TestEntities.java

```diff
@@ -54,10 +55,14 @@
 		@Column(name = "")
 		boolean booleanField;
 
+		// This long is forced to be stored as a String for testing
+		@Column(columnTypeCode = TypeCode.STRING)
 		long longField;
 
 		double doubleField;
 
+		// This double array is forced to be stored as a String for testing
+		@Column(columnTypeCode = TypeCode.STRING)
```

Comment:
What happens if user marks a property to be stored in incompatible column type? Like property of type double[] to be stored in TypeCode.BOOL
Should have a test for that?

Reply:
We can't stop them from doing this, and what would happen is that during mapping/conversion we would get errors saying there isn't a converter provided that can handle the situation, and we do have explicit exceptions thrown with helpful messages for that case.

Reply:
added a test!